applications/crossbar/priv/couchdb/schemas/system_config.jonny5.json

```diff
@@ -11,7 +11,7 @@
         },
         "default_{Key}": {
             "description": "jonny5 default {Key}",
-            "type": "integer"
+            "type": "boolean"
```

Comment:
This should actually be purged from the system config doc. Until we can infer Key, I think it better to leave these off the doc. Thoughts?